features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -327,6 +337,7 @@ Feature: Collection
   # mod can view signups
   When I log out
     And I am logged in as "mod1"
+    And all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.